src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateBuildOptionsRule.cs

```diff
@@ -355,5 +355,15 @@ private IncludeContext GetCopyToOutputIncludeContext(CommonCompilerOptions compi
                     null,
                     ProjectFilesCollection.DefaultPublishExcludePatterns);
         }
+
+        private string FormatLanguageVersion(string langVersion)
+        {
+            if (langVersion.StartsWith("csharp"))
```

Comment:
Migration only migrates to csharp, there shouldn't be any cases where we're migrating a different value unless you can think of something.